tuf-spec.md

```diff
@@ -687,10 +687,10 @@ repo](https://github.com/theupdateframework/specification/issues).
 
 * **4.4. File formats: snapshot.json**
 
-   The snapshot.json file is signed by the snapshot role.  It lists the version
-   numbers of all metadata on the repository, excluding timestamp.json and
-   mirrors.json.  For the root role, the hash(es), size, and version number
-   are listed.
+   The snapshot.json file is signed by the snapshot role. It lists the version
+   numbers, and optionally the size, of all metadata on the repository,
+   excluding root.json, timestamp.json and mirrors.json. For delegated roles,
```

Comment:
Hashes SHOULD be OPTIONAL for the top-level and all delegated targets roles.

Reply:
No, hashes are optional now for the top-level targets and delegated targets roles. You might like to list hashes for a particular use case (see Section 5.6).